src/main/java/net/atos/zac/app/ontkoppeldedocumenten/converter/RESTOntkoppeldDocumentConverter.java

```diff
@@ -41,7 +42,15 @@ public RESTOntkoppeldDocument convert(final OntkoppeldDocument document) {
         return restDocument;
     }
 
-    public List<RESTOntkoppeldDocument> convert(final List<OntkoppeldDocument> documenten) {
-        return documenten.stream().map(this::convert).collect(Collectors.toList());
+    public List<RESTOntkoppeldDocument> convert(
+            final List<OntkoppeldDocument> documenten,
+            final List<UUID> informatieobjectTypeUUIDs
+    ) {
+        return IntStream.range(0, documenten.size())
```

Comment:
Same here. Why not just loop with a forEachIndexed over the documents? In Kotlin at least that's easily done with `forEachIndexed`.

Reply:
I would prefer to have Kotlin conversion in another PR, cause @infonl-marcel is relying on the changes in this one for the frontend.